design-proposals/network-binding-plugin/network-binding-plugin.md

```diff
@@ -189,6 +189,7 @@ the binding is referenced by name with optional additional arguments.
         - domainAttachment (a standard domain definition that exists in
           the core . E.g `tap`, `sriov`).
         - downwardAPI (E.g `device-info`)
+        - resourceOverhead (currently only additional memory overhead request for the compute container in the virt-launcher pod is supported)
```

Comment:
I would be specific here and mention the compute-container in the name.
E.g. `computeResources`.

This also opens up the different questions and options which should be discussed in a proposal:
- Should we consider only the compute container? What about the sidecar container?
- Ref the compute container, we can use a specific name for the field (e.g. `computeResources`) or we can make it configurable using a flag under a general `resources`.
- What is the reason for not just doing a `computeMemoryRequest` needs to convince the reviewers.

Now, we can make it even more general by moving away from the specific network usage of this and look at a general sidecar hook that KubeVirt supports.
E.g.: The Kubvirt CR will have a policy for allocating resources to sidecar containers and possible the compute container. Referencing this policy from the network binding definition or from a sidecar hook. This may be an overkill, but thinking about it and examining the pors/cons of it can be useful.